config/templates/chef-marketplace-cookbooks/non_root_solo.rb.erb

```diff
@@ -0,0 +1,8 @@
+cookbook_path '/opt/chef-marketplace/embedded/cookbooks'
+cache_path '/tmp/chef-marketplace/local-mode-cache'
```

Comment:
If we do that then we have to change the umask on /var/opt/chef-marketplace to be open for writes by non-privileged users and I'm not so sure we want to do that.